main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
         float countTip(float tip);
     };
 
-    //1) serve food
     void serveFood();
-    //2) collect payment
-    float collectPayement(float payment);
-    //3) provide eating utensils
+    float collectPayment(float payment);
     void utensils();
 };
```

Comment:
I didn't catch these improper function names in part 1. 
What you have written here does not match your plain-english actions.
You need to include the action words in your function names
```
void provideEatingUtensils();
```
- [ ] revise